internal/handlers/pg/msg_insert.go

```diff
@@ -128,7 +132,7 @@ func (h *Handler) insertMany(ctx context.Context, sp *pgdb.SQLParam, docs *types
 }
 
 // insert prepares and executes actual INSERT request to Postgres.
-func (h *Handler) insert(ctx context.Context, sp *pgdb.SQLParam, doc any) error {
+func (h *Handler) insert(ctx context.Context, dbPool *pgdb.Pool, sp *pgdb.SQLParam, doc any) error {
```

Comment:
If we pass `dbPool` as param, should method `func (h *Handler) insert` be just `func insert` then?

Reply:
Good point, will do in the next PR